### PR TITLE
Wrap parcel-query in a function

### PR DESCRIPTION
**Pull request by thebriando**
# ↪️ Pull Request
We're creating a CLI that is using `parcel-query` and we need to have it wrapped in a function to make it easier to import.

## 🚨 Test instructions

Run `parcel-query`

## ✔️ PR Todo

- [ ] Added/updated unit tests for this change
- [ ] Filled out test instructions (In case there aren't any unit tests)
- [ ] Included links to related issues/PRs

### CODE REVIEW

File: packages/dev/query/src/cli.js [hunk 1]
```diff
@@ -17,769 +17,787 @@ import {Priority} from '@parcel/core/src/types';
...
+        'stats',
+        {
+          help: 'Statistics',
+          action: stats,
+        },
+      ],
+      [
+        'findAsset',
+        {
+          help: 'args: <regex>. Lsit assets matching the filepath regex',
```
Comment (mischnic) [inline packages/dev/query/src/cli.js]:
```suggestion
          help: 'args: <regex>. List assets matching the filepath regex',
```

**Review by mischnic — APPROVED** [review]

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 1.50s | -3.00ms |
| Cached | 235.00ms | -44.00ms 🚀 |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/index.b8ae99ba.css | 94.00b | +0.00b | 244.00ms | -16.00ms 🚀 |
| dist/modern/index.31cedca9.css | 94.00b | +0.00b | 243.00ms | -17.00ms 🚀 |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 4.14s | -71.00ms |
| Cached | 373.00ms | -14.00ms |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/PermalinkedComment.b41e2791.js | 4.07kb | +0.00b | 421.00ms | +55.00ms ⚠️ |
| dist/UserProfile.2a2fa310.js | 1.51kb | +0.00b | 422.00ms | +56.00ms ⚠️ |
| dist/NotFound.13a965e5.js | 399.00b | +0.00b | 422.00ms | +56.00ms ⚠️ |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 38.18s | -393.00ms |
| Cached | 2.30s | +104.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/media-viewer.6b8adda1.js | 537.45kb | +0.00b | 12.53s | +3.36s ⚠️ |
| dist/card.583a1f18.js | 140.31kb | +0.00b | 8.73s | -438.00ms 🚀 |
| dist/archive.a2217524.js | 60.29kb | +0.00b | 12.53s | +3.36s ⚠️ |
| dist/component-lazy.1b33c14d.js | 59.63kb | +0.00b | 6.45s | +819.00ms ⚠️ |
| dist/ConfigPanelFieldsLoader.3cdf42ed.js | 15.95kb | +0.00b | 8.73s | -437.00ms 🚀 |
| dist/codeViewerRenderer.f99075be.js | 2.74kb | +0.00b | 12.35s | +3.18s ⚠️ |
| dist/zh.fcdc32bb.js | 1.96kb | +0.00b | 8.73s | -437.00ms 🚀 |
| dist/ro.a6eff34a.js | 612.00b | +0.00b | 8.62s | +2.31s ⚠️ |
| dist/index.html | 240.00b | +0.00b | 12.61s | +6.30s ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/card.583a1f18.js | 140.31kb | +0.00b | 10.75s | +2.37s ⚠️ |
| dist/pdfRenderer.187ba54d.js | 12.21kb | +0.00b | 10.77s | +2.39s ⚠️ |
| dist/mobile-upload.f2b8a53f.js | 7.99kb | +0.00b | 10.77s | +2.39s ⚠️ |
| dist/codeViewerRenderer.f99075be.js | 2.74kb | +0.00b | 10.77s | -804.00ms 🚀 |
| dist/pl.5f36d63e.js | 2.38kb | +0.00b | 7.43s | +1.24s ⚠️ |
| dist/pt_BR.eccfad73.js | 2.19kb | +0.00b | 7.43s | +1.24s ⚠️ |
| dist/ko.2cf2bbda.js | 2.11kb | +0.00b | 7.43s | +1.24s ⚠️ |
| dist/nb.b300dd3e.js | 2.09kb | +0.00b | 7.43s | +1.24s ⚠️ |
| dist/nl.3999ea58.js | 2.07kb | +0.00b | 7.43s | +1.24s ⚠️ |
| dist/pt_PT.402f9c4e.js | 765.00b | +0.00b | 7.43s | +1.24s ⚠️ |
| dist/simpleHasher.38195df7.js | 719.00b | +0.00b | 10.75s | +2.37s ⚠️ |
| dist/index.html | 240.00b | +0.00b | 6.20s | -5.44s 🚀 |

</p></details>

<details><summary>Three.js ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 3.21s | +25.00ms |
| Cached | 316.00ms | -8.00ms |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

*No bundle changes detected.*

</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/371801094550979141)**

File: packages/dev/query/src/run.js
Comment (lettertwo) [inline packages/dev/query/src/run.js:1]: Could this file be eliminated by just moving its contents into `bin.js`?

Comment (thebriando) [reply packages/dev/query/src/run.js:1]: Sure

**Pull request merged**